src/App.js

```diff
@@ -12,11 +12,12 @@ import UserProfile from './components/user/UserProfile';
 import CheckoutComplete from './components/Purchase/CheckoutComplete';
 import PageNotFound from './components/PageNotFound';
 import AlbumChat from './components/album/AlbumChat';
+import RecommendationPage from './components/RecommendationPage';
 import ForgotPassword from './components/ForgotPassword';
 import { createTheme, CssBaseline, ThemeProvider } from '@mui/material';
 
 function App() {
-  const { status } = useAuth();
+  const { status, user } = useAuth();
```

Comment:
This will work fine, but I would suggest instead fetching the current user within the `RecommendationPage` component. My logic being that the logic around what the component wants to render and how it wants to handle having a logged-in user or not all belongs to that component.

Reply:
Akos , i was thinking that we want to have recommendations based on user logged in and his history, album purchased etc, or may be I am mistaken? Sorry I didn't catch up

Reply:
Yes that's correct; my previous comment was regarding getting the `user` in `App.js` to pass in a user id to the component. My suggestion was to instead have `const { status, user } = useAuth()` inside the `RecommendationPage` itself. But it's only a small optimization, so I'm okay with leaving it as-is.

Reply:
Akos , do you mean to fetch user in this way?  commit 660ec8671459cc367a4a5be452cc6da2d5b1c71f

Reply:
Yep exactly; so we should be able to remove this change here in `App.js`, and no longer need to `RecommendationPage` component on line 147

Reply:
I have removed this route in App,js commit f02e07058c3c647521957fb92caaf3a4c0347f26
Now I have eslint error that user assigned a value but never used in line 20 in App.js
`const { status, user } = useAuth();`
we used user in deleted route
Should I remove it as well?

Reply:
Just the `user` part can be removed. 

> , and no longer need to `RecommendationPage` component on line 147

It looks like I mistyped above ^. I did not mean that you should remove the route altogether, just remove the use of `user` attribute/prop in that route component. Apologies for the confusion.

So in the end App.js should look like:

```javascript
function App() {
  const { status } = useAuth();
  const isLoggedIn = status === AuthStatus.LoggedIn;
  
  // etc all the existing code
  
  return (
  	// existing code, existing routes, then the recommendation one, probably right under the user profile one
  	<Route
          path="/profile/recommendations"
          element={
            <RequireAuth>
              <RecommendationPage />
            </RequireAuth>
          }
    />
  )
}
```

Reply:
Thank you ❤️ , added, commit b7c70b6b4b564cfa8198230d5f3378742bf489c3